README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Consider adding troubleshooting tips if the installation fails or if the server does not start as expected.